config/commands/general.ts

```diff
@@ -116,3 +116,13 @@ export const vocalCount = {
     count: 'Il y a {count} personne{plural} connectée{plural} dans ce salon vocal !',
   },
 };
+
+export const mergePDF = {
+  options: {
+    aliases: ['mergepdf', 'mergePDF', 'pdf'],
```

Comment:
```suggestion
    aliases: ['mergepdf', 'pdf'],
```
Les aliases sont déjà case-insensitive :)